configuration/configuration_reference.md

```diff
@@ -601,6 +678,39 @@ Two users would be in the role 'pipeline-operators', they are **Jez** and **lqia
 
 [top](#top)
 
+## &lt;pluginRole&gt; {#plugin_role_definition}
+
+The `<pluginRole>` element is used to define roles in GoCD. Unlike `role` which contains a list of `users`, `pluginRole` provides configuration to map a GoCD role to a role defined in an external authorization server managed by an Authorization plugin. e.g pluginRole can be used to define mappings between LDAP group and GoCD roles.
+
+**Notes:**
+
+| Attribute | Required | Description |
+|-----------|----------|-------------|
+| name | Yes | The name of the role. |
+| authConfigId | Yes | Plugin-id of an authorization plugin. |
```

Comment:
We should add a link to `<authConfig>`. The `authConfigId` is not a `plugin id`, it is the id of `authConfig`